disco_aws_automation/disco_vpc_peerings.py

```diff
@@ -26,81 +29,95 @@ class DiscoVPCPeerings(object):
     """
     This class takes care of processing of a VPC's peering connections
     """
-    def __init__(self, vpc, boto3_ec2):
-        self.disco_vpc = vpc
-        self.boto3_ec2 = boto3_ec2
 
-    def update_peering_connections(self, dry_run=False):
+    def __init__(self, boto3_ec2=None):
+        if boto3_ec2:
+            self.client = boto3_ec2
+        else:
+            self.client = boto3.client('ec2')
+
+    def update_peering_connections(self, vpc, dry_run=False, delete_extra_connections=False):
         """ Update peering connections for a VPC """
-        desired_peerings = self.parse_peering_strs_config(self.disco_vpc.environment_name)
-        existing_peerings = self._get_existing_peerings()
+        desired_peerings = self._get_peerings_from_config(vpc.get_vpc_id())
+        existing_peerings = self._get_existing_peerings(vpc)
 
         logger.info("Desired VPC peering connections: %s", desired_peerings)
         logger.info("Existing VPC peering connections: %s", existing_peerings)
 
-        if existing_peerings > desired_peerings:
+        if delete_extra_connections and existing_peerings > desired_peerings:
```

Comment:
What's the greater-than operator doing in this context?  Is it acting as the superset operator?